draft-ietf-masque-connect-ip.md

```diff
@@ -1046,6 +1048,33 @@ References:
 : This document
 {: spacing="compact"}
 
+## Creation of the MASQUE URI Suffixes Registry {#iana-suffix}
+
+This document requests that IANA create a new "MASQUE URI Suffixes" registry
+maintained at IANA_URL_TBD. This new registry contains three columns:
```

Comment:
I think the general description here need to be clear that it is for the suffixes following the well known "masque" suffix and include reference to https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml

Reply:
Agreed, fixed